CMakeLists.txt

```diff
@@ -4,12 +4,11 @@ project (tunnler CXX)
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-set(RAKNET_PREFIX "${CMAKE_CURRENT_SOURCE_DIR}/externals/RakNet")
-include_directories(${RAKNET_PREFIX}/source)
-
+set(RAKNET_PREFIX "${PROJECT_SOURCE_DIR}/externals/RakNet")
+include_directories(${RAKNET_PREFIX}/Source)
 add_subdirectory(${RAKNET_PREFIX})
 
-include_directories("${PROJECT_SOURCE_DIR}/include" ${RAKNET_INCLUDE_DIR}/raknet)
+include_directories("${PROJECT_SOURCE_DIR}/include")
```

Comment:
The old one was probably worse, but `${RAKNET_INCLUDE_DIRS}` does actually exist in the raknet cmake